lxd/auth/oidc/oidc.go

```diff
@@ -505,6 +505,41 @@ func (o *Verifier) startSession(ctx context.Context, w http.ResponseWriter, idTo
 	return nil
 }
 
+// getSecretFromUsedAtTime returns the secret that should have been used to encrypt a cookie or sign a token at the
+// given unix time in seconds. A boolean is returned to indicate if the returned secret is not the most recent, this
+// prompts the Verifier to refresh the session token, so that it doesn't become unverifiable.
+func (o *Verifier) getSecretFromUsedAtTime(ctx context.Context, usedAtTimeUnixSeconds int64) (*cluster.AuthSecret, bool, error) {
```

Comment:
what about naming the bool return type "needsRefresh" rather than indicating its most recent, to make it clearer to the caller what they need to do?